packages/tiny-router/src/index.ts

```diff
@@ -1,13 +1,15 @@
 import TinyRouter from "../types";
 import { set, init, beforeSSR } from "./actions";
+import WPSource from "@frontity/wp-source/src";
 
 const tinyRouter: TinyRouter = {
   name: "@frontity/tiny-router",
   state: {
     router: {
       link: "/",
       autoFetch: true
-    }
+    },
+    source: WPSource().state.source // TODO: notr sure if that's correct
```

Comment:
It's not correct. The state from other packages is available on runtime.

Reply:
gotcha 👍